backend/v3/domain/domain.go

```diff
@@ -61,67 +66,59 @@ type domainChanges interface {
 	SetUpdatedAt(t time.Time) database.Change
 }
 
-// import (
-// 	"math/rand/v2"
-// 	"strconv"
-
-// 	"github.com/zitadel/zitadel/backend/v3/storage/cache"
-// 	"github.com/zitadel/zitadel/backend/v3/storage/database"
-
-// 	// "github.com/zitadel/zitadel/backend/v3/telemetry/logging"
-// 	"github.com/zitadel/zitadel/backend/v3/telemetry/tracing"
-// 	"github.com/zitadel/zitadel/internal/crypto"
-// )
-
-// // The variables could also be moved to a struct.
-// // I just started with the singleton pattern and kept it like this.
-// var (
-// 	pool              database.Pool
-// 	userCodeAlgorithm crypto.EncryptionAlgorithm
-// 	tracer            tracing.Tracer
-// 	// logger            logging.Logger
-
-// 	userRepo func(database.QueryExecutor) UserRepository
-// 	// instanceRepo func(database.QueryExecutor) InstanceRepository
-// 	cryptoRepo func(database.QueryExecutor) CryptoRepository
-// 	orgRepo    func(database.QueryExecutor) OrgRepository
-
-// 	// instanceCache cache.Cache[instanceCacheIndex, string, *Instance]
-// 	orgCache cache.Cache[orgCacheIndex, string, *Org]
-
-// 	generateID func() (string, error) = func() (string, error) {
-// 		return strconv.FormatUint(rand.Uint64(), 10), nil
-// 	}
-// )
-
-// func SetPool(p database.Pool) {
-// 	pool = p
-// }
+// The variables could also be moved to a struct.
+// I just started with the singleton pattern and kept it like this.
+var (
+	pool database.Pool
+	// userCodeAlgorithm crypto.EncryptionAlgorithm
+	tracer tracing.Tracer
+	logger logging.Logger
+
+	// userRepo func(database.QueryExecutor) UserRepository
+	// instanceRepo func(database.QueryExecutor) InstanceRepository
+	// cryptoRepo func(database.QueryExecutor) CryptoRepository
+	orgRepo func(database.QueryExecutor) OrganizationRepository
```

Comment:
it is a wrapper for the org repo constructor. So for testing we could also pass this function into the `CommandOpts` and allow overwriting it in the opts.